### ISSUE TITLE
ChromeCast support?

### ISSUE THREAD

**Issue by OutRunner** [sprylab/texturevideoview#18]
Hi,

Do you know if there are any known issues/restrictions to using TextureVideoView in a ChromeCast app? I'm writing a presenter app that displays images, text and videos and I'm using TextureVideoView for the videos but they're not displaying anything on the ChromeCast. I can hear definitely the video playing but the view itself remains a blank box. It's definitely working when playing videos in a regular Activity on my tablet device, it's just when using a CastPresentation it isn't working.

BTW TextureVideoView is a fantastic component, I was having so many problems with videos before finding it.

Cheers!

**Comment by rzimmer** [comment]
Glad you like TextureVideoView! 😄 
Unfortunately, we don't have any experience using it with ChromeCast. I'd expected that it just works. Do you know if Android's original VideoView works correctly?
I could imagine that there are issues with TextureView and ChromeCast, but I really don't know, though. You might do a bit of research in this direction and report back. Maybe that helps us sorting the issue out....

**Comment by OutRunner** [comment]
After a fair bit of experimentation I've discovered that actually TextureVideoView does work with ChromeCast! The problem is with my test devices (two Lenovo Tab 2 A10-30s), for some reason they refuse to send any video to ChromeCast including when using the regular VideoView. 

I tried my Nexus 7 and another cheap unbranded tablet I've got and they both work, so this appears to be a bug in Lenovo's implementation of video playback. 

Thanks for your help and sorry for the false alarm!

Cheers!
